src/pmt.c

```diff
@@ -1921,15 +1921,18 @@ int process_sdt(int filter, unsigned char *sdt, int len, void *opaque) {
     for (i = 11; i < sdt_len - 1; i += desc_loop_len) {
         b = sdt + i;
         sid = b[0] * 256 + b[1];
+        status = b[3] >> 4;
         desc_loop_len = (b[3] & 0xF) * 256 + b[4];
         desc_loop_len += 5;
         pmt = get_pmt_for_sid(ad->id, sid);
-        LOGM("Detected service ID %d (%X), pos %d len %d", sid, sid, i,
-             desc_loop_len);
+        LOGM("Detected service ID %d (%X) status:%d CA:%d, pos %d len %d", sid, sid,
+             (status >> 1),(status & 0x1), i, desc_loop_len);
         if (!pmt) {
             LOG("%s: no PMT found for sid %d (%X)", __FUNCTION__, sid, sid);
             continue;
         }
+        //if (opts.clean_psi)  // TODO: Patch the SDT table
```

Comment:
The PR looks good, however please remove the comments

Reply:
Done! Ready to merge. 😉